### ISSUE TITLE
cc: error: unrecognized command line option ‘-no-pie’

### ISSUE THREAD

**Issue by Sod-Almighty** [soemraws/cl-libusb#4]
Cannot load libusb-ffi:

```
* (load "~/quicklisp/setup.lisp")
T
* (ql:quickload :libusb-ffi)
To load "libusb-ffi":
  Load 1 ASDF system:
    libusb-ffi
; Loading "libusb-ffi"
; pkg-config libffi --cflags

; cc -o /root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel-tmpGHU3ALSV.o -c -g -Wall -Wundef -Wsign-compare -Wpointer-arith -O3 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wunused-parameter -fno-omit-frame-pointer -momit-leaf-frame-pointer -fno-pie -fPIC -I/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/ /root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel.c
/root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel.c: In function ‘main’:
/root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel.c:12:7: warning: unused variable ‘autotype_tmp’ [-Wunused-variable]
   int autotype_tmp;
       ^
; cc -o /root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel-tmpAAURSO1 -g -Wl,--export-dynamic -no-pie /root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel.o
cc: error: unrecognized command line option ‘-no-pie’
While evaluating the form starting at line 21, column 0
  of #P"/root/quicklisp/dists/quicklisp/software/gsll-quicklisp-eeeda841-git/gsll.asd":

debugger invoked on a ASDF/FIND-SYSTEM:LOAD-SYSTEM-DEFINITION-ERROR in thread #<THREAD "main thread" RUNNING {10005305B3}>: Error while trying to load definition for system gsll from pathname /root/quicklisp/dists/quicklisp/software/gsll-quicklisp-eeeda841-git/gsll.asd: Subprocess #<UIOP/LAUNCH-PROGRAM::PROCESS-INFO {1003470D13}>
 with command ("cc" "-o" "/root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel-tmpAAURSO1" "-g" "-Wl,--export-dynamic" "-no-pie" "/root/.cache/common-lisp/sbcl-1.5.2-linux-x64/root/quicklisp/dists/quicklisp/software/cffi_0.20.1/libffi/libffi-types__grovel.o")
 exited with error code 1
```
Using SBCL 1.5.2, ASDF 3.3.1 and quicklisp 2019-02-16 on a Debian 8 64-bit system (kernel 3.16.0-8-amd64).

**Comment by soemraws** [comment]
I'm not sure if this is a libusb-ffi error. It seems that the compiler has this problem while compiling the libffi-types file in libffi.

I would advise you to first see if you can quickload cffi and cffi-grovel without issues, and/or possibly ask the cffi project for help on this error.

**Comment by Sod-Almighty** [comment]
Yup, I can load both of those without issue:
```
* (load "~/quicklisp/setup.lisp")

T
* (ql:quickload :cffi)
To load "cffi":
  Load 1 ASDF system:
    cffi
; Loading "cffi"
......
(:CFFI)
* (ql:quickload :cffi-grovel)
To load "cffi-grovel":
  Load 1 ASDF system:
    cffi-grovel
; Loading "cffi-grovel"

(:CFFI-GROVEL)
*
```

**Comment by soemraws** [comment]
This issue is closed in favor of issue #5.
